corehq/apps/locations/permissions.py

```diff
@@ -321,3 +321,19 @@ def _inner(request, domain, loc_id, *args, **kwargs):
         return location_restricted_response(request)
 
     return require_can_edit_or_view_locations(_inner)
+
+
+def can_access_location_data(view_fn):
+    """
+    Decorator controlling a user's access to VIEW data for a specific location.
+    The decorated function must be passed a loc_id arg (eg: from urls.py)
```

Comment:
should this doc be updated as you are fetching the loc_id within the function it self.?

Reply:
Done